apps/sms/views/conversation/test/unit/compose_test.js

```diff
@@ -1127,16 +1127,20 @@ suite('compose_test.js', function() {
         assert.isTrue(sendButton.disabled);
       });
 
-      test('enabled if both lock and Compose allow sending', function() {
```

Comment:
umm, i removed this because, now compose always allow sending. There is no check on compose now, as we can send empty message also.